sync/class.jetpack-sync-defaults.php

```diff
@@ -271,18 +283,6 @@ class Jetpack_Sync_Defaults {
 		'site-logo',
 	);
 
-	static function is_whitelisted_option( $option ) {
```

Comment:
Do we know why this method was here, if not in use?  I hesitate to take it out of another plugin we release was somehow depending on it.